src/openms_gui/source/VISUAL/APPLICATIONS/GUITOOLS/TOPPView.cpp

```diff
@@ -141,6 +141,17 @@ int main(int argc, const char** argv)
 
   try
   {
+#if defined(__APPLE__)
+    // see https://bugreports.qt.io/browse/QTBUG-104871
+    // if you link to QtWebEngine and the corresponding macros are enabled, it will
+    // try to default to OpenGL 4.1 on macOS (for hardware acceleration of WebGL in Chromium, which we do not need yet)
+    // but our OpenGL code for 3D View is written in OpenGL 2.x.
+    // Now we force 2.1 which is also available on all? Macs.
+    QSurfaceFormat format;
+    format.setVersion(2, 1); // the default is 2, 0
```

Comment:
why is default 2.0 but setting to 2.1 solbes the issue with the default being 4.1? confused :)

Reply:
please see the linked Qt issue and related code. Too much to explain.

Reply:
Feel free to comment on the issue to ask for more background info

Reply:
it's fine - no questions asked :P guess it just falls back to 4.1 for everything != 4.1 or 2.1

Reply:
Webengine forces 3.2+
macOS only has 2.1 and 4.1
So with Webengine on macOS it defaults to 4.1. We cannot do that, so we have to go to 2.x
Since macOS only has 2.1, we go 2.1